functions-and-operators/json-functions.md

```diff
@@ -70,14 +70,19 @@ TiDB supports most of the [JSON functions](https://dev.mysql.com/doc/refman/8.0/
 | [JSON_ARRAYAGG(key)](https://dev.mysql.com/doc/refman/8.0/en/aggregate-functions.html#function_json-arrayagg) | Provides an aggregation of keys. |
 | [JSON_OBJECTAGG(key, value)](https://dev.mysql.com/doc/refman/8.0/en/aggregate-functions.html#function_json-objectagg) | Provides an aggregation of values for a given key. |
 
+## Validation functions
+
+| Function Name                     | Description |
+| --------------------------------- | ----------- |
+| [JSON_SCHEMA_VALID(schema,json_doc)](https://dev.mysql.com/doc/refman/8.0/en/json-validation-functions.html#function_json-schema-valid) | Validate a JSON document against a schema. |
```

Comment:
```suggestion
| [JSON_SCHEMA_VALID(schema,json_doc)](https://dev.mysql.com/doc/refman/8.0/en/json-validation-functions.html#function_json-schema-valid) | Validate a JSON document against a schema to improve data integrity and consistency. |
```

Reply:
Updated in f198d36b8833f4dd60f7a4b8ad26083a84f5fb6e 

I don't think "to improve" is correct here as the function only checks the data and doesn't change (improve) the data.

PTAL @hfxsd